stellar/deploy-contract.js

```diff
@@ -50,6 +50,14 @@ async function getInitializeArgs(config, chain, contractName, wallet, options) {
         case 'interchain_token_service': {
             const gatewayAddress = nativeToScVal(Address.fromString(chain?.contracts?.axelar_gateway?.address), { type: 'address' });
             const gasServiceAddress = nativeToScVal(Address.fromString(chain?.contracts?.axelar_gas_service?.address), { type: 'address' });
+            const hubChain = config.chains.axelar || config.axelar;
```

Comment:
Just wondering, why do we need to consider both cases?

Reply:
We don't. This is a mistake